Userland/Libraries/LibWeb/HTML/Scripting/ExceptionReporter.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2022, David Tuin <davidot@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/TypeCasts.h>
+#include <LibJS/Runtime/VM.h>
+#include <LibJS/Runtime/Value.h>
+#include <LibWeb/HTML/Scripting/ExceptionReporter.h>
+
+namespace Web::HTML {
+
+void report_exception(JS::ThrowCompletionOr<JS::Value> const& result)
```

Comment:
Needs a spec link: https://html.spec.whatwg.org/#report-the-exception

Reply:
Whoops didn't know it was an actual spec. I didn't really follow the spec so I added a FIMXE.
(Also oh oh: `The existence of both [report an error](https://html.spec.whatwg.org/#report-the-error) and [report an exception](https://html.spec.whatwg.org/#report-the-exception) is confusing, and both algorithms have known problems. You can track future cleanup in this area in [issue #958](https://github.com/whatwg/html/issues/958).`)